docs/examples/201/api-management-create-all-resources/main.bicep

```diff
@@ -0,0 +1,254 @@
+  name: '${apiManagementService.name}/exampleUser2'
+  properties: {
+    firstName: 'ExampleFirstName2'
+    lastName: 'ExampleLastName2'
+    email: 'examplefirst2@example.com'
+    state: 'active'
+    note: 'note for example user 2'
+  }
+}
+
+resource exampleUser3 'Microsoft.ApiManagement/service/users@2020-06-01-preview' = {
+  name: '${apiManagementService.name}/exampleUser1'
+  properties: {
+    firstName: 'ExampleFirstName3'
+    lastName: 'ExampleLastName3'
+    email: 'examplefirst3@example.com'
+    state: 'active'
+    note: 'note for example user 3'
+  }
+}
+
+resource exampleProperty 'Microsoft.ApiManagement/service/properties@2019-01-01' = {
+  name: '${apiManagementService.name}/exampleproperties'
+  properties: {
+    displayName: 'propertyExampleName'
+    value: 'propertyExampleValue'
+    tags: [
+      'exampleTag'
+    ]
+  }
+}
+
+resource subscription1 'Microsoft.ApiManagement/service/subscriptions@2017-03-01' = {
+  name: '${apiManagementService.name}/examplesubscription1'
+  properties: {
+    displayName: 'examplesubscription1'
+    productId: exampleProduct.id
+    userId: exampleUser1.id
+  }
+}
+
+resource subscription2 'Microsoft.ApiManagement/service/subscriptions@2017-03-01' = {
+  name: '${apiManagementService.name}/examplesubscription2'
+  properties: {
+    displayName: 'examplesubscription2'
+    productId: exampleProduct.id
+    userId: exampleUser3.id
+  }
+}
+
+resource certificate 'Microsoft.ApiManagement/service/certificates@2020-06-01-preview' = {
+  name: '${apiManagementService.name}/exampleCertificate'
+  properties: {
+    data: mutualAuthenticationCertificate
+    password: certificatePassword
+  }
+}
+
+resource exampleGroup 'Microsoft.ApiManagement/service/groups@2020-06-01-preview' = {
+  name: '${apiManagementService.name}/exampleGroup'
+  properties: {
+    displayName: 'Example Group Name'
+    description: 'Example group description'
+  }
+}
+
+resource openIdConnectProvider 'Microsoft.ApiManagement/service/openidConnectProviders@2020-06-01-preview' = {
+  name: '${apiManagementService.name}/exampleOpenIdConnectProvider'
+  properties: {
+    displayName: 'exampleOpenIdConnectProviderName'
+    description: 'Description for example OpenId Connect provider'
+    metadataEndpoint: 'https://example-openIdConnect-url.net'
+    clientId: 'exampleClientId'
+    clientSecret: openIdConnectClientSecret
+  }
+}
+
+resource exampleLogger 'Microsoft.ApiManagement/service/loggers@2020-06-01-preview' = {
+  name: '${apiManagementService.name}/exampleLogger'
+  properties: {
+    loggerType: 'azureEventHub'
+    description: 'Description for example logger'
+    credentials: {
+      name: 'exampleEventHubName'
+      eventHubNamespaceConnectionString: eventHubNamespaceConnectionString
+    }
+  }
+}
+
+resource identityProvider 'Microsoft.ApiManagement/service/identityProviders@2020-06-01-preview' = {
+  name: 'google'
```

Comment:
is this name correct? it doesn't have to be prefixed by the name of the parent resource?

Reply:
I think you're right but Bicep does not let me prefix with the service name.

<img width="935" alt="Capture d’écran 2020-12-14 à 20 15 16" src="https://user-images.githubusercontent.com/5141735/102124943-3359f400-3e49-11eb-851f-30b0596ef97e.png">
<img width="709" alt="Capture d’écran 2020-12-14 à 20 13 53" src="https://user-images.githubusercontent.com/5141735/102124966-394fd500-3e49-11eb-84ae-3250774d66bc.png">

Reply:
Solved!